docs/search/criteria_reference/contentname_criterion.md

```diff
@@ -0,0 +1,41 @@
+# ContentName Criterion
+
+The [`ContentName` Search Criterion](https://github.com/ibexa/core/blob/main/src/contracts/Repository/Values/Content/Query/Criterion/ContentName.php)
+searches for content by its name.
+
+## Arguments
+
+- `value` - string(s) representing the Content name(s), the joker `*` can be used for partial search.
```

Comment:
```suggestion
- `value` - string(s) representing the Content name(s), the wildcard character `*` can be used for partial search.
```

It's the first time I see the `joker` name - I thought that wildcard is the common term?

See:
https://learn.microsoft.com/en-us/sql/t-sql/language-elements/percent-character-wildcard-character-s-to-match-transact-sql?view=sql-server-ver16
https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-wildcard-query.html
https://solr.apache.org/guide/8_7/the-standard-query-parser.html#wildcard-searches
https://en.wikipedia.org/wiki/Wildcard_character